src/build/helpers.ts

```diff
@@ -4,6 +4,7 @@ import * as Browser from "./types.js";
 export const bufferSourceTypes = new Set([
   "ArrayBuffer",
   "SharedArrayBuffer",
+  "ArrayBufferLike",
```

Comment:
This is for Web IDL keywords and not expected to include TS specific things. This should go elsewhere.